extensions/mongodb-client/deployment/pom.xml

```diff
@@ -63,8 +63,8 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-metrics-deployment</artifactId>
-            <scope>test</scope>
+            <artifactId>quarkus-micrometer-registry-prometheus-deployment</artifactId>
```

Comment:
Can you make it with just the `quarkus-micrometer` extension?

Reply:
@brunobat I get:
```
[ERROR] Rule 4: io.quarkus.enforcer.BansRuntimeDependency failed with message:
[ERROR] 1 illegal runtime dependencies found that have to be replaced with their -deployment counterparts:
[ERROR]         quarkus-micrometer
```

Reply:
you need the `quarkus-micrometer-deployment`

Reply:
I mean, replace `quarkus-micrometer-registry-prometheus-deployment` with `quarkus-micrometer-deployment`.
I'm not sure if it will cause you an error on the tests, if so `quarkus-micrometer-registry-prometheus-deployment` can be there but as a test dependency.
The reason is that the `quarkus-micrometer-registry-prometheus-deployment` is a particular telemetry output lib for prometheus, there are other implementations like otlp and we should avoid depending directly on them o the main code.

Reply:
Makes sense. I did exactly that. Still need to fix imports sorting. Then it should be ok

Reply:
Thanks!